README.md

```diff
@@ -1,24 +1,16 @@
-# Lily watchdog
+# SaffronStatus
```

Comment:
I'd like to to be formatted `Saffron Status` or for the repo probably `saffron-status`. I'm not a big fan of PascalCase for this one, but maybe I'm being overly picky, you tell me.